mirrord/cli/src/main.rs

```diff
@@ -404,6 +405,43 @@ async fn register_to_waitlist(email: EmailAddress) -> Result<()> {
     Ok(())
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, Default)]
+struct VerifiedConfig {
+    config: Option<String>,
+    errors: Option<String>,
```

Comment:
make config an object, right now make it return the target
make errors a Vec, and also add warnings.

Reply:
target = pod / deployment / rollout and/or namespace

Reply:
Returns the `TargetConfig` right?

Reply:
It doesn't have to be `TargetConfig` exactly - it needs to be what the extension needs - which is namespace if any and pod/deploymento r any (so it can know if it should show a dialog)

Reply:
But isn't this the same thing?

Reply:
I am just saying that it's up to you - we might not want it to be bound by TargetConfig since if we add stuff there it'll be propagated back here, and we might be better with other approach.

Reply:
Alright, I get it now.

tbh I'm not quite sure what the IDE side would want from this, so maybe leaving it as the `TargetConfig` is best? I don't really want to introduce a struct in the middle here, where we just have to convert for no reason, while we don't add any crazy stuff that would break this.